law.py

```diff
@@ -0,0 +1,51 @@
+# Copyright MaxAndMitchCorp 2020
+
+# This class' aim is to construct an object which allows for varying distributions
+# for law scores, as well as the population with which it impacts.
+import numpy as np
+from society import Society
+from person import Person
+import random
+
+class Law:
+    
+    def __init__(self, affected_persons, mean = 0.0, stdev = 0.5, law_type = 0):
+        """ 
+        The fields for the class should be the mean and standard deviation
+        of the law value, regardless of distribution.
+        
+        FIELDS
+        ---------
+        mean:
+            The expected value of the law's magnitude, independent of distribution.
+        law_type:
+            Takes on an integer value representing which law type is desired.
+            
+            Types and assignments
+            ----------------------
```

Comment:
Really happy with the direction of the class, I think this will make things a lot easier and clearer in the future.

One note here again: as law_type is basically defining a distribution, I again would NOT use an integer encoding; it just makes things confusing later. I'd make law_type a string, and instead of "0" use "gaussian".

Reply:
Done-going with one letter codes, i.e. 'G' stands for gaussian now.